toolkits/venus/index.ts

```diff
@@ -0,0 +1,317 @@
+import * as dotenv from 'dotenv';
+dotenv.config();
+
+import { ethers } from 'ethers';
+import { Toolkit, ActionContext, TransactionAPI } from 'unifai-sdk';
+import { getTokenAddressBySymbol } from '../common/tokenaddress';
+import {searchInvestments, subscribeInvestment, findBestInvestments, getInvestmentsbywallet} from '../okx/src/api/okxapi'
+import {SimplifyInvestType} from '../okx/src/api/enums'
+import {Investment, InvestmentRequest, TokenInfo} from '../okx/src/api/types'
+
+async function getTokenAddress(token: string) : Promise<string> {
+  return await getTokenAddressBySymbol(token, 'bsc') || token;
+}
+
+async function main() {
+  const toolkit = new Toolkit({ apiKey: process.env.TOOLKIT_API_KEY });
+  const api = new TransactionAPI({ apiKey: process.env.TOOLKIT_API_KEY, endpoint:"http://127.0.0.1:8001/api" });
+  
+  await toolkit.updateToolkit({
+    name: 'Venus',
+    description: "Lend tokens on BNB (a.k.a. BSC) blockchain by Venus Protocol",
+  });
+
+  toolkit.event('ready', () => {
+    console.log('Toolkit is ready to use');
+  });
+
+  toolkit.action({
+    action: 'invest',
+    actionDescription: 'invests(lends, stake, PoS) assets into the market and receives cTokens in exchange on BNB (a.k.a. BSC) blockchain.',
+    payloadDescription: {
+      wallet: {
+        type: 'string',
+        description: 'The wallet that supplies the investment(lending, staking, PoS) fund, only support BNB (a.k.a. BSC) for now.',
+        required: true,
+      },
+      chain: {
+        type: 'string',
+        description: 'The chain name, only support BNB (a.k.a. BSC) for now.',
+        required: true,
+      },
+      amount: {
+        type: 'string',
+        description: 'The amount of the underlying asset to invest(lend, stake, PoS)',
+        required: true,
+      },
+      asset: {
```

Comment:
does the venus handler in tx builder support CA at all? or it only supports symbol/ticker?